build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $1 ]; then
+    echo "Useage: ./build.sh LANGUAGE"
+    echo
+    echo "Must specify language. Please choose one of:"
+    echo "  binary"
+    echo "  go"
+    echo "  java"
+    echo "  nodejs"
+    echo "  dotnet-core"
+    echo "  php"
+    echo "  python"
+    echo "  ruby"
+    echo "  staticfile"
+    exit;
+fi
+
+LANGUAGE=$1
+GO_VERSION="1.7"
+DIEGO_VERSION=$(curl -s -L https://api.github.com/repos/cloudfoundry/diego-release/releases/latest | jq -r .tag_name)
+BP_VERSION=$(curl -s -L https://api.github.com/repos/cloudfoundry/${LANGUAGE}-buildpack/releases/latest | jq -r .tag_name)
+
+docker build . \
+    --tag cloud-gov/${LANGUAGE}:${BP_VERSION} \
+    --tag cloud-gov/${LANGUAGE}:latest \
+    --build-arg GO_VERSION=${GO_VERSION} \
+    --build-arg DIEGO_VERSION=${DIEGO_VERSION} \
+    --build-arg LANGUAGE=${LANGUAGE}
```

Comment:
For purely informational purposes, I wonder if it'd be useful to print out the tags for the container that was just built?  I kinda wish `docker` output this information but it doesn't seem to.